### PR TITLE
Simplify chart - use new local no migration

### PR DESCRIPTION
**Pull request by nopcoder**
- Use new local kv implementation, no local postgres
- Remove the migration kv as the implementation in this version do not migrate from postgres.

NOTE: When data migration as part of KV will be enabled we should include a job that will perform the work - but for now there is no need to submit any job.

### CODE REVIEW

File: charts/lakefs/README.md [hunk 1]
```diff
@@ -29,13 +30,35 @@ To install the chart with custom configuration values:
...
+      region: us-east-1
+  gateways:
+    s3:
+      domain_name: s3.lakefs.example.com
+```
+
+Example `my-values.yaml` using DynamoDB:
+```yaml
+secrets:
+  databaseConnectionString: postgres://postgres:myPassword@my-lakefs-db.rds.amazonaws.com:5432/lakefs?search_path=lakefs
```
Comment (johnnyaug) [inline charts/lakefs/README.md:54]: Needed for dynamo?

File: charts/lakefs/README.md [hunk 2]
```diff
@@ -46,13 +69,17 @@ lakefsConfig: |
...
-You should give your Kubernetes nodes access to all S3 buckets you intend to use lakeFS with.
-If you can't provide such access, lakeFS can be configured to use an AWS key-pair to authenticate (part of the `lakefsConfig` YAML below).
+You should give your Kubernetes nodes access to all S3 buckets (or other resources) you intend to use lakeFS with.
+If you can't provide such access, lakeFS can be configured to use an AWS key-pair to authenticate (part of the `lakefsConfig` YAML).
 
 ## Notable Chart Upgrades
 
+### Upgrading from chart version 0.7.XX or lower
+
+Database configuration is not set to use PostgreSQL by default using environment
```
Comment (johnnyaug) [inline charts/lakefs/README.md:81]:
```suggestion
If you are using Postgres as your database, make sure your `lakefsConfig` property contains the key `database.type` and that it is set to `postgres`. Before this version, the Helm chart set this property implicitly.
```

**Review by johnnyaug — CHANGES_REQUESTED** [review]

File: charts/lakefs/README.md (hunk 1 above)
Comment (nopcoder) [reply charts/lakefs/README.md:54]: For dynamo as we did for the s3 bucket - we didn't add any specific secret. Dynamo users the same credentials use to access AWS bucket.

Comment (nopcoder) [reply charts/lakefs/README.md]: Removed the use of connection string with dynamo db example

**Pull request merged**